### DISCUSSION TITLE
Pengiriman gagal

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#155] [Tanya Jawab]
selemat siang mas rio,

mau tanya apakah apikirimwa sedang trobel kah ?
muncul pesan seperti ini saat pengiriman WA.

![image](https://user-images.githubusercontent.com/88919468/203914448-e0a36e2a-bfbf-472e-ad2e-94a80b045a04.png)

**Comment by newfaboc** [comment]
sudah bisa mas rio.

Terima kasih.

**Reply by masbagusstwn** [reply]
kalo error biasanya 500 atau 503 .. kalo 502 biasanya ada masalah sama url bukan?
